iop/network/smb2man/src/COPYING

```diff
@@ -0,0 +1,24 @@
+Libsmb2 components fall under two separate licences
+
+
+The lib and include directories
+===============================
+The smb2 client library itself, i.e. the lib and include directories,
+is licenced under GNU Lesser General Public License as published by
+the Free Software Foundation; either version 2.1 of the License, or
+(at your option) any later version.
+
+A copy of LGPL 2.1 is included in the file LICENCE-LGPL-2.1.txt but can also be downloaded from http://www.gnu.org/licenses/old-licenses/lgpl-2.1.html
```

Comment:
I think this specific license can't be included here...

Reply:
Could use libsmb2 as external then?

Reply:
Because if use the code as internal it should not conflict inside.

Reply:
I’m not expert here, but I would say that this is not even possible to be external…

Reply:
I will close it, there's no chance of merge it, if anyone wants run smb2man on ps2sdk it will be hard to do it properly without the conflicts of L-GPL